br/pkg/task/stream.go

```diff
@@ -1243,9 +1244,12 @@ func restoreStream(
 		} else {
 			totalDureTime := time.Since(startTime)
 			summary.Log("restore log success summary", zap.Duration("total-take", totalDureTime),
-				zap.Uint64("restore-from", cfg.StartTS), zap.Uint64("restore-to", cfg.RestoreTS),
+				zap.Uint64("restore-from-tso", cfg.StartTS),
+				zap.Uint64("restore-to-tso", cfg.RestoreTS),
+				zap.Uint64("RestoreTS-tso", currentTS),
```

Comment:
I feel like we need mention the upstream/downstream cluster for user to easy understand.
for example. `restore-from` and `restore-to` were belong to upstream cluster.
Maybe call them `upstream-start-point` and `upstream-end-point`. `downstream-end-point`  or something like that?

Reply:
![image](https://github.com/pingcap/tidb/assets/1272980/6c19fbb5-1925-40be-859a-d117e5655147)

I think we should aim to use similar names for the restore of a full backup and for the restore based on pitr.

I like think the `-{start,end}-point` names are much better than `-tso`.

Instead of upstream/downstream, I think we should use source and target.

Another difference with a regular restore is that this message returns a timestamp in uint64 and human readable form while a regular restore only has the uint64 version.

Reply:
@3pointer I have updated the names. PTAL

Reply:
it looks better now.